.gitbook/1-users/3-data-exchange/2-provide-data.md

```diff
@@ -1 +1,74 @@
-# Consume data
+# Provide data
+
+Data providers can provide their data that match the requirements of the data deal and in return earn a reward.
+
+The provider request verification of the data to oracle, and submits consent with oracle signature to the Panacea public blockchain.
+
+Since the data is encrypted throughout the whole process of data transmission, no one can get the original data.
+
+The transmission of data and the payment of rewards are managed atomically through the Panacea public blockchain, so consumers can get the data only when the reward payment is completed.
+
+## Post data to oracle
+
+Encrypt data that you wants to provide. 
+Encryption is done using your chain account key and oracle public key.
+This makes only oracles can decrypt your original data.
+
+You can encrypt data by the following CLI:
+```bash
+panacead encrypt-data ${input-file-path} ${your-account-key-name}\
```

Comment:
```suggestion
panacead encrypt-data ${input-file-path} ${your-account-key-name} \
```
It's generally better to put space in front of backticks, so that any kind of shells can execute it without any unexpected problems.